x/ibc/genesis_test.go

```diff
@@ -44,17 +44,17 @@ func (suite *IBCTestSuite) TestValidateGenesis() {
 				),
 				ConnectionGenesis: connection.NewGenesisState(
 					[]connection.ConnectionEnd{
-						connection.NewConnectionEnd(ibctypes.INIT, connectionID, clientID, connection.NewCounterparty(clientID2, connectionID2, commitmenttypes.NewMerklePrefix([]byte("prefix"))), []string{"1.0.0"}),
+						connection.NewConnectionEnd(common.INIT, connectionID, clientID, connection.NewCounterparty(clientID2, connectionID2, commitmenttypes.NewMerklePrefix([]byte("prefix"))), []string{"1.0.0"}),
```

Comment:
nit: these enum values should have prefixes. if someone comes to look at this code they will not know what you are Initing here. Optimize for the reader, not writer

Reply:
They have the prefix and they are using a custom name

Reply:
hmm, I meant that `INIT` here should be something like `STATE_INIT`. the proto type is correct but for someone going through the go type like here it is unclear what is being initialized.